tools/scripts/generic.mk

```diff
@@ -204,6 +211,7 @@ CFLAGS += -Wall								\
 #------------------------------------------------------------------------------
 #                          COMMON INITIALIZATION
 #------------------------------------------------------------------------------
+relative_to_project = $(addprefix $(PROJECT_BUILD)/,$(call get_relative_path,$1))
```

Comment:
nitpick: It seems after update you didn't put it in the same place

Reply:
i put it intentionately to this place because I'm pretty sure it needs PROJECT_BUILD to be defined
and PROJECT_BUILD is now defined in [platform].mk just above where I placed it

Reply:
It seems better here then.
Anyway, as a note. If `:=` is used (Ex 'var2 := $(VAR1)`)  then $(VAR1) is evaluated otherwise (with just `=`, just the reference is used and it is evaluated later when actually needed.